Automattic/wp-calypso
# Plans step redesign: Setup experiment and display header

#### Changes proposed in this Pull Request

This PR sets up the experiment for changing the signup plans step design. Users in the treatment group will see the new header title for the plans step.

**Important links**
1. [Figma design](https://www.figma.com/file/gnYBUOBthTMckJRJr3pZmk/Control-Experiments?node-id=1362%3A813)
2. pau2Xa-2yE-p2 for the project thread.

**Header title if a custom domain is selected**:

<img width="597" alt="Screenshot 2021-02-18 at 9 03 16 PM" src="https://user-images.githubusercontent.com/1269602/108380488-e3beec80-722c-11eb-9456-26a09f60459f.png">

**Header title if a free subdomain is selected**:

<img width="845" alt="Screenshot 2021-02-18 at 9 03 37 PM" src="https://user-images.githubusercontent.com/1269602/108380527-ef121800-722c-11eb-9ff0-3b8768b054f4.png">

#### Testing instructions



* Open http://calypso.localhost:3000/ and assign yourself to `treatment` of the `signup_plans_step_redesign_v1` experiment in the Abacus tool.
* Head to /start and go through the signup flow. At the plans step of signup, verify that you see the new title as shown in the screenshots above. Verify for both free subdomain and custom domain selections.
